src/Tvheadend.h

```diff
@@ -242,6 +242,16 @@ class CHTSPDemuxer
   {
     return (time_t)m_timeshiftStatus.shift;
   }
+  inline time_t GetTimeshiftBufferStart() const
+  {
+    // Note: start/end mismatch is not a bug. tvh uses inversed naming logic here!
+    return static_cast<time_t>(m_timeshiftStatus.end);
```

Comment:
I'm not sure casting to `time_t` is correct here. While working on some refactoring locally I changed the method above these to return `int64_t` instead. Considering tvheadend reports these values as microseconds it's quite easy to end up truncating the value if casting to a 32-bit value at this stage.